### DISCUSSION TITLE
import com.nozbe.watermelondb.jsi.WatermelonJSI is not available in android packages

### DISCUSSION THREAD

**Discussion by fullstackNRJ** [Nozbe/WatermelonDB#1286] [Q&A]
Hey @radex, Thanks for the amazing work with watermelondb, looking forward to contributing to it.

I need help with turbo syncing using native code, I came across this in the docs. 
<img width="755" alt="image" src="https://user-images.githubusercontent.com/17597033/159156894-3eb4e2be-3839-44ff-9876-ac89df2935df.png">

However, I am unable import `jsi` package and `WatermelonJSI` class because it's missing. I checked the packages in android studio and I cannot find `@nozbe/watermelondb/native/android-jsi, there's only android counterpart.

How can I install this package or jar as they call in java world. or if there's a similar method available that I can use from available classes ?

Basically, I'm just trying to prepopulate one of the tables in watermelon db with someExternal.db file on app launch.

RN version : "0.67.1"
Watermelondb version : "^0.24.0"

P.S : I'm new to react-native and specially native android side of the world. Any sort of help much appreciated. ../\..

**Comment by KrisLau** [comment]
Might help 

- https://github.com/Nozbe/WatermelonDB/issues/1216#issuecomment-980800458
